packages/suite/src/actions/recovery/recoveryActions.ts

```diff
@@ -68,22 +68,14 @@ const checkSeed = () => async (dispatch: Dispatch, getState: GetState) => {
         dispatch(setError(response.payload.error));
     }
 
-    if (device.features.recovery_mode) {
-        TrezorConnect.getFeatures({
-            device: {
-                path: device.path,
-            },
-        });
-    }
-
     dispatch(setStatus('finished'));
 };
 
 const recoverDevice = () => async (dispatch: Dispatch, getState: GetState) => {
     const { advancedRecovery, wordsCount } = getState().recovery;
     const { device } = getState().suite;
     if (!device || !device.features) return;
-
+    dispatch(setError(''));
```

Comment:
why?

Reply:
starting new call erases previous error and results either in finished state which may or may not add new error.